package.json

```diff
@@ -79,12 +80,14 @@
     "react": "^16.8.3",
     "react-dnd-test-backend": "^7.4.4",
     "react-test-renderer": "^16.8.5",
+    "regenerator-runtime": "^0.13.2",
```

Comment:
What is this for and should it be a dev-dependency?

Reply:
this is standalone runtime for Regenerator-compiled generator and async functions, uses for solving `regeneratorRuntime is not defined` error while testing code which depends on @cognite/3-viewer. Needed only for testing purposes. Will move to setupFile for jest.config